translation from c00/30/16 to 20 (#66)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/17/1/translation.txt

//
// Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
//

> 一番見られたくない方に、見つかるとは……みっともない姿をお見せして申し訳ございません。=I'm deeply sorry to appear as such... In front of the person I least wanted to see me like this.

Reviewer: ```suggestion
一番見られたくない方に、見つかるとは……みっともない姿をお見せして申し訳ございません。=Discovered by the last person I'd want to find me... I'm ashamed you found me in this state.
```

Author: my non-english brain kind of gets the idea that she has something against the guy if there's no other information. (girl is at neutral affection from the rule of thumb. 
how would you feel about: "Discovered by the last person I wanted to see me like this... I'm ashamed you found me in this state."
or "Discovered by the last person I'd want to see me like this... I'm ashamed you found me in this state." if you want to keep the would there

Reviewer: I chose the wording so it was ambiguous if it was because she dislikes the player, or because she's does like the player and so is embarrassed. But either of your suggestions is fine.

Author: went with first suggestion then.

っ……こんな時にもよおすとは……恥の上塗りです……=Tsk... It's utterly deplorable... To need to pee at a time like this...

> ……間抜けな姿になっていることは充分自覚しておりますので、あまり見ないでいただけると助かります。=...I'm fully aware of how stupid I look, so I would appreciate it if you didn't stare too much.

Reviewer: This is ok, but trying to sound more natural
```suggestion
……間抜けな姿になっていることは充分自覚しておりますので、あまり見ないでいただけると助かります。=I know I look... stupid.  Please don't stare.
```

Author: small letter after "..." ? until now i used upper case every time after it, even if it's the same sentence. i could just avoid it with "i know i look stupid.... [...]. i'll defer to your opinion, but i am curious if i should keep using upper case after "..." or if i can go with lower case too.

Reviewer: It's one sentence. In this case `...` is like a `,`:  "I` know I look [pause] stupid"

I think it should be: if it's the same sentence, don't capitalize. If it's a new sentence, then capitalize.  @SpockBauru: oppinion?

> ここへきて尿意が……私はとうとう天から見放されたようです……=I can feel the urge to pee... It looks like my time has run out...

Reviewer: I think the first part is more either
- "It's happening right now": `I'm peeing myself...`
- "It's going to happen in a second": `I'm going to wet myself...`

While the MTL for the second part sounds silly, it's probably closer. Maybe: `The gods have forsaken me...` or `My luck has run out...`

Author: well, it seems like she is just getting the urge, and realizing she's in dire straights, since she can't get out. so the event has not yet happened. 
and yes, already mentioned it in discord, that mtl was so funny i really wanted to keep it, if you think it works too, i'll definitely go with it :D   (and, ofc, luck was one of the variants i thought as well, but it felt even more remote :P )

Reviewer: I think the idea of the gods turning away and luck running out are equivalent here

……恥を忍んでお願い申し上げます。私を助けていただけませんか？=...I'll swallow my pride and ask. Could you please help me?

> ……すみません。理由を明かすことはできませんが、一刻も早く助けていただけないでしょうか。お願い致します。=...Excuse me. I can't reveal the reason, but could you please help me as soon as possible? Thank you.

Reviewer: ```suggestion
……すみません。理由を明かすことはできませんが、一刻も早く助けていただけないでしょうか。お願い致します。=...Excuse me. I can't explain, but could you please help me as soon as possible? Thank you.
```
or

```suggestion
……すみません。理由を明かすことはできませんが、一刻も早く助けていただけないでしょうか。お願い致します。=...Excuse me. Quickly, I can't explain, but could you please help me? Thank you.
```

Author: does using quickly after "...excuse me" work? it feels a bit odd to start like that and then suddenly be urgent. maybe move it at the end, "help me quickly"? again, if you say it works at the start, i don't really have anything against it

Reviewer: No, that's the correct way someone would start when they **need** help from someone they don't want to ask for help from

Author: roger that! done

> 犯しまくる絶好の機会だ、そういったことをお考えなのでしょうね。=I guess you're probably thinking this is a great opportunity to attack me, right?

Reviewer: ```suggestion
犯しまくる絶好の機会だ、そういったことをお考えなのでしょうね。=You're probably thinking this is the perfect opportunity to have your way with me, right?
```

Could probably loose the `, right?`, but whatever you think.

Author: how about ", i bet." or maybe even "aren't you?". she's actually egging him on, girl is in slavery mode.

Reviewer: Just leave the `, right?` then I think

Author: ok, left it

> 一つご報告します、我慢の限界がきて失禁してしまいそうです。失禁プレイがお嫌いでなければ、このままでも構いませんが……？=I would like to report that I've reached the limit of my patience and incontience. But if this is the sort of play you like, I guess you could keep me like this?

Reviewer 2: ```suggestion
一つご報告します、我慢の限界がきて失禁してしまいそうです。失禁プレイがお嫌いでなければ、このままでも構いませんが……？=I would like to report that I've reached the limit of my patience and incontience. But if this is the sort of play you like, I guess you could keep me like this.
```

Author: here i'd keep the ? (actually, i think ...? would fit even better, especially since it's in the jp line too), since it's just her being hesitant/uncertain.

> 一つご報告します、我慢の限界がきて失禁してしまいそうです。失禁プレイがお嫌いでなければ、このままでも構いませんが……？=I would like to report that I've reached the limit of my patience and incontience. But if this is the sort of play you like, I guess you could keep me like this...?

Reviewer: ```suggestion
一つご報告します、我慢の限界がきて失禁してしまいそうです。失禁プレイがお嫌いでなければ、このままでも構いませんが……？=Just so you know, I'm about to lose control of my bladder...  But if that doesn't bother you, should we keep going...?
```
